scene/resources/default_theme/default_theme.cpp

```diff
@@ -982,7 +982,7 @@ void make_default_theme(bool p_hidpi,Ref<Font> p_font) {
 	Ref<BitmapFont> default_font;
 	if (p_font.is_valid()) {
 		default_font=p_font;
-	} if (p_hidpi) {
+	} else if (p_hidpi) {
```

Comment:
I think the bug is elsewhere. With this change the check for p_font.is_valid() becomes useless. If we don't need it remove the whole case.